R/eml.R

```diff
@@ -521,60 +544,84 @@ eml_individual_name <- function(given_names=NULL, sur_name) {
   indiv_name
 }
 
-
 #' Create an eml-project section.
 #'
-#' Note: This is super-limited right now.
+#' Note - studyAreaDescription, designDescription, and relatedProject are not fully fleshed out. Need to pass these objects in directly if you want to use them.
 #'
-#' @param title (character) Title of the project.
-#' @param awards (character) One or more awards for the project.
-#' @param first (character) First name of the person with role `role`.
-#' @param last (character) Last name of the person with role `role`.
-#' @param organizations (character) Optional. One or more organization strings.
-#' @param role (character) Optional. Specify an alternate role.
+#' @param title (character) Title of the project (Required).
+#' @param personnelList (list of eml_personnel) Personnel involved with the project (role:originator required).
```

Comment:
Should be "(list of personnel").
Remove "(role:originator required)." since we found out this is not correct.